core/designsystem/src/main/java/com/iteneum/designsystem/components/TextField.kt

```diff
@@ -167,3 +171,70 @@ fun LpOutlinedTextFieldNumber(
         keyboardOptions = KeyboardOptions.Default.copy(keyboardType = KeyboardType.Number)
     )
 }
+
+/**
+ * [DropdownTextField] it's a textfield to show a list of items inside a box
+ *
+ * @param title refers to the label that the textfield will have
+ * @param items refers to the list that will be given to expand the textfield
+ * @param modifier to set component modifier
+ * @param
```

Comment:
You forget to add the param selected, please add the description for this parameter